yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/DigitalIdentityController.java

```diff
@@ -83,6 +84,11 @@ public String identityShareComplete(Model model) throws URISyntaxException {
         LOG.info("Requesting Share QR code creation for session with ID '{}'", sessionId);
         ShareSessionQrCode sessionQrCode = createShareQrCode(sessionId);
 
+        String qrCodeId = sessionQrCode.getId();
+
+        LOG.info("Requesting Share QR code with ID '{}'", qrCodeId);
```

Comment:
This log is repeated in `DigitalIdentityService.fetchShareQrCode` 
```
LOG.info("Requesting Share Session QR code with ID '{} at '{}'", qrCodeId, path);
```
so it is probably not needed here. 

does it have to be INFO, it seems like a DEBUG level log

Same thing for the log above: (I know this one wasn't added by you)
```
LOG.info("Requesting Share QR code creation for session with ID '{}'", sessionId);
```